src/pipeline_config.py

```diff
@@ -3,7 +3,7 @@
 from attrdict import AttrDict
 from deepsense import neptune
 
-from utils import read_params
+from .utils import read_params
 
 ctx = neptune.Context()
 params = read_params(ctx, fallback_file='neptune.yaml')
```

Comment:
@taraspiotr not sure if this will work in `python run ...` under current structure (I suppose 'src/neptune.yaml' should).